### ISSUE TITLE
Loss is always zero.

### ISSUE THREAD

**Issue by hassheh** [liuzhejun/ResNet-for-Radio-Recognition#1]
I have tried to replicate the ResNet_Model.ipynb with the exact same dataset and network.
While training the model, the loss is always zero after each epoch. Is there something I am missing?

/device:GPU:0
Train on 503999 samples, validate on 216001 samples
Epoch 1/100
 - 555s - loss: 0.0000e+00 - val_loss: 0.0000e+00
